### PR TITLE
feat: Add remove EXIF button

### PR DESCRIPTION
**Pull request by SladeGranger**
**What this PR does？**

添加是否去除图片EXIF选项的按钮。
后端修改的对应pr为: https://github.com/halo-dev/halo/pull/1889

### CODE REVIEW

**Comment by JohnNiang** [comment]
Hold this PR until upstream PR https://github.com/halo-dev/halo/pull/1889 got merged.
/hold

File: src/views/system/optiontabs/AttachmentTab.vue [hunk 1]
```diff
@@ -10,6 +10,9 @@
       <a-form-model-item label="上传图片时预览：">
         <a-switch v-model="options.attachment_upload_image_preview_enable" />
       </a-form-model-item>
+      <a-form-model-item label="是否去除图片EXIF：">
```
Comment (ruibaby) [inline src/views/system/optiontabs/AttachmentTab.vue:13]:
```suggestion
      <a-form-model-item label="去除图片 EXIF 信息：">
```

**Review by ruibaby — CHANGES_REQUESTED** [review]
/hold

等待 https://github.com/halo-dev/halo/pull/1889 完成合并。

**Review by JohnNiang — COMMENTED** [review]
> https://github.com/halo-dev/halo/pull/2168 已经合并

/lgtm

**Review by ruibaby — COMMENTED** [review]
/approve

**Review by ruibaby — APPROVED** [review]

**Pull request merged**